Purchases/Public/RCPurchases.m

```diff
@@ -684,19 +692,23 @@ - (void)syncPurchasesWithReceiptRefreshPolicy:(RCReceiptRefreshPolicy)refreshPol
 - (void)handleRestoreReceiptPostWithInfo:(RCPurchaserInfo *)info
                                    error:(NSError *)error
                     subscriberAttributes:(RCSubscriberAttributeDict)subscriberAttributes
-                              completion:(RCReceivePurchaserInfoBlock)completion {
+                              completion:(nullable RCReceivePurchaserInfoBlock)completion {
```

Comment:
this is called from a method that has a nullable completion

Reply:
Ooops